src/main/java/org/mule/extension/sftp/internal/connection/SftpClient.java

```diff
@@ -204,7 +204,7 @@ private void checkExists(String path) {
   }
 
   private void connect() throws IOException {
-    org.apache.sshd.sftp.client.SftpClientFactory scf = org.apache.sshd.sftp.client.SftpClientFactory.instance();
+    SftpConcurrentClientFactory scf = SftpConcurrentClientFactory.INSTANCE;
```

Comment:
You could add the instance() method to preserve the way that it gets to that instance.

Reply:
Done